CondFormats/GEMObjects/src/GEMeMap.cc

```diff
@@ -79,7 +79,7 @@ void GEMeMap::convertDummy(GEMROMapping& romap) {
   uint8_t gebId = 0;
 
   for (int re = -1; re <= 1; re = re + 2) {
-    for (int st = GEMDetId::minStationId; st <= GEMDetId::maxStationId; ++st) {
+    for (int st = 0; st <= GEMDetId::maxStationId; ++st) {
```

Comment:
Why not ```GEMDetId::minStationId0```, since you have it already defined in DataFormats/MuonDetId/interface/GEMDetId.h?

Reply:
done